Presentation.Web/Controllers/API/V2/External/ItSystems/ItSystemV2Controller.cs

```diff
@@ -71,6 +71,7 @@ public class ItSystemV2Controller : ExternalBaseController
         /// <param name="numberOfUsers">Greater than or equal to number of users filter</param>
         /// <param name="includeDeactivated">If set to true, the response will also include deactivated it-interfaces</param>
         /// <param name="changedSinceGtEq">Include only changes which were LastModified (UTC) is equal to or greater than the provided value</param>
+        /// <param name="usedInOrganizationUuid">Uuid of the organization containing systems in use</param>
```

Comment:
rephrase:
`Filter by UUID of an organization which has taken the it-system into use through an it-system-usage resource`